src/groupeddataframe/splitapplycombine.jl

```diff
@@ -414,10 +416,45 @@ julia> combine(df -> sum(df.c), gd) # Slower variant
 
 See [`by`](@ref) for more examples.
 """
-function combine(f::Any, gd::GroupedDataFrame)
+combine(f::Function, gd::GroupedDataFrame; keepkeys::Bool=true) =
+    combine_helper(f, gd, keepkeys=keepkeys)
+combine(gd::GroupedDataFrame, f::Function; keepkeys::Bool=true) =
+    combine_helper(f, gd, keepkeys=keepkeys)
+
+function DataFrames.combine(gd::GroupedDataFrame, cs::Pair...; keepkeys::Bool=true)
+    if any(x -> first(x) isa Tuple, cs)
+        Base.depwarn("passing a Tuple as column selector is deprecated, use " *
+                     "an AbstractVector instead", :combine)
```

Comment:
Can you print the syntax directly? That should be more useful in 90% of cases.

Reply:
fixed